dashboard/app/controllers/docs_proxy_controller.rb

```diff
@@ -0,0 +1,29 @@
+require 'net/http'
+require 'set'
+
+# This proxy controller allows us to serve curriculumn builder docs that live on
+# docs.code.org as if they live on studio.code.org. This is done so that we can
+# render them in an iframe, and access that iframe without having cross-origin problems
+class DocsProxyController < ApplicationController
+  include ProxyHelper
+
+  ALLOWED_HOSTNAME_SUFFIXES = %w(
+    docs.code.org
+  )
+
+  EXPIRY_TIME = 30.minutes
+
+  def get
+    # request.original_url will look something like http://studio.code.org/docs/csd/maker_leds/index.html
+    # We want to redirect them to https://docs.code.org/csd/maker_leds/index.html
+    docs_route = request.original_url.split('docs')[1..-1].join
```

Comment:
Not sure if this is the best way to extract this route. Originally did `params[:docs_route]` but that didn't get me the ".html" portion. I could use a regex instead of this, but not sure it makes a difference.